src/client/components/ImageMenu.js

```diff
@@ -139,71 +166,61 @@ ImageMenu.propTypes = {
 }
 
 const getBorder = props => {
-  let color
-
-  if (props.isDragAccept) color = '#00e676'
-  else if (props.isDragReject) color = '#ff1744'
-  else if (props.isDragActive) color = '#2196f3'
-  else if (props.dragging) color = '#446fe9'
-  if (color) return `2px ${color} dashed`
-  else return '2px white solid'
+  if (props.isDragAccept) return `2px #00e676 dashed;`
+  if (props.isDragReject) return `2px #ff1744 dashed;`
+  if (props.isDragActive) return `2px #2196f3 dashed;`
+  if (props.dragging) return `2px #446fe9 dashed;`
+
+  return `1px solid rgba(209, 209, 209, ${getBackgroundAlpha(props)});`
 }
 
 const isVisible = props =>
   props.isDragAccept ||
   props.isDragReject ||
   props.isDragActive ||
-  props.dragging
+  props.dragging ||
+  props.active
 
-const getVisibility = props => {
-  if (isVisible(props)) return 'visible'
-  else return 'hidden'
-}
-const getOpacity = props => {
-  if (isVisible(props)) return '0.8'
-  else return '0'
-}
+const getVisibility = props => (isVisible(props) ? 'visible' : 'hidden')
+
+const getBackgroundAlpha = props => (isVisible(props) ? '0.8' : '0')
```

Comment:
Nice one-liners! Thanks. I think we can clean these functions just a little more by removing the wrapping parentheses. Example:

    const getBackgroundAlpha = props => isVisible(props) ? '0.8' : '0'

Reply:
Yes, but eslint/prettier has different opinion. Actually it's the second time eslint has problem with something that can be allowed, but `prettier` decides to be strict about it. What do we do in such cases? I tried adding:
`"no-extra-parens": [2, "all"]`
but it conflicts with prettier and both cases are highlighted, with and without parentheses. I found no quick way of telling prettier to acknowledge that if there is no problem, there is really no problem.
I'm going to leave it for now.

Reply:
Oh, I didn't know we had prettier configured here. In that case, I'll abide by the rule, ""no code style discussion, ever. add linter rule or stfu." 

🤐